federation-js/src/service/__tests__/printCoreSchema.test.ts

```diff
@@ -20,66 +20,78 @@ describe('printCoreSchema', () => {
   });
 
   it('produces a parseable output', () => {
+    try {
+      parse(coreSchema);
+    } catch (e) {
+      console.log(JSON.stringify(e));
```

Comment:
Why the change to manually logging the error instead of the `.not.toThrow()` expectation?

Reply:
It still had the expectation, just needed to clean this up.
37773db